src/test/resources/application-test.conf

```diff
@@ -0,0 +1,49 @@
+akka {
+  stdout-loglevel = "OFF"
+  loglevel = "DEBUG"
+  use-slf4j = on
+  management {
+    http {
+      hostname = "0.0.0.0"
+      bind-hostname = 0.0.0.0
+      route-providers-read-only = false
+    }
+    health-checks {
+      readiness-checks {
+        ready = "it.pagopa.interop.attributeregistryprocess.server.impl.HealthCheck"
+      }
+      liveness-checks {
+        live = "it.pagopa.interop.attributeregistryprocess.server.impl.LiveCheck"
+      }
+    }
+  }
+}
+
+attribute-registry-process {
+  port = 18088
+  jwt {
+    audience = "aud1"
+  }
+  services {
+    attribute-registry-management = "http://localhost:8086/attribute-registry-management/0.0"
+    agreement-process = "http://localhost:8086/agreement-process/0.0"
+    agreement-management = "http://localhost:8086/agreement-management/0.0"
+    catalog-management = "http://localhost:8086/catalog-management/0.0"
+    tenant-management = "http://localhost:8086/tenant-management/0.0"
```

Comment:
[same](https://github.com/pagopa/interop-be-attribute-registry-process/pull/1/files#r1135229414)